README.md

```diff
@@ -19,34 +19,101 @@ In addition to this README file, there are docs covering the following topics:
 * [Design](docs/design.md)
   * Describes the technical design and implementation details of the Version Control app
 
-## Local Dev & Test Environment
+## Installation
+
+### Installation Considerations
+
+There are some special considerations for running the Version Control app:
+
+* Nautobot 1.1.0 or later is required
+* The Nautobot installation **must** be running a Dolt database
+* There are some [additional configurations](#configuring-nautobot-to-use-version-control) required in `nautobot_config.py`
+
+The version control app can be installed with pip3:
+
+```no-highlight
+pip3 install git+https://github.com/nautobot/nautobot-plugin-version-control
+```
+
+To ensure the version control app is automatically reinstalled during future upgrades, create a new file named `local_requirements.txt` (if not already existing) in the Nautobot root directory (e.g. `/opt/nautobot`) to include the `nautobot-plugin-version-control` package:
+
+```no-highlight
+echo nautobot-plugin-version-control >> local_requirements.txt
```

Comment:
If we're publishing to PyPI, the `pip3 install` command should just be `pip3 install nautobot-plugin-version-control`; if we're not publishing to PyPI, then the local_requirements.txt won't work as given.

Reply:
Yeah we will be. Good catch, thanks.